src/prefect/client/schemas/objects.py

```diff
@@ -560,6 +563,11 @@ class FlowRun(ObjectBaseModel):
         description="A list of tags on the flow run",
         examples=[["tag-1", "tag-2"]],
     )
+    labels: KeyValueLabels = Field(
+        default_factory=dict,
+        description="Prefect Cloud: A dictionary of key-value labels. Values can be strings, numbers, or booleans.",
```

Comment:
labels are only available via cloud atm so adding them onto client schema here with a default dict to populate for non-cloud server responses so that their available to be serialized as span attrs